src/internal/server/PrintLeftoverLabelController_test.go

```diff
@@ -0,0 +1,78 @@
+package server_test
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"src/internal/server"
+	"testing"
+)
+
+type testParams_PrintLeftoverLabelHandler struct {
+	reqMethod          string
+	reqBody            io.Reader
+	expectedStatusCode int
+	expectedMessage    string
+}
+
+// TODO: add tests once the implementation for PrintLeftoverLabelController is complete
+// range of test cases to iterate
+var testParams = []testParams_PrintLeftoverLabelHandler{}
+
+func TestPrintLeftoverLabelController(t *testing.T) {
+
+	testCount := len(testParams)
```

Comment:
Leave a comment to explain what this is for

Reply:
I decided to remove it altogether; I don't think it's possible that we would skip a request without `t.Fail()` being called.